run.js

```diff
@@ -3,7 +3,7 @@ import express from 'express'
 import { readFileSync } from 'node:fs'
 
 const signer = new ethers.Wallet(readFileSync('signing.key').toString('hex'))
-console.log(`Using signer account ${await signer.getAddress()}`)
+signer.getAddress().then((address) => console.log(`Using signer account: ${address}`))
```

Comment:
I don't think this change does anything semantically different? But I'm fine with t.

Reply:
Yeah, this is needed for building a binary from the code using npx pkg. It doesn't like an `await` call here

Reply:
That's a bit strange - it is maybe ignoring the type = module? Anyway, this is fine as a workaround.